src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -586,11 +573,6 @@ def simulated_test_runner():
                     gamecontroller,
                 )
 
-            # Only validate on the blue worlds
-            blue_full_system_proto_unix_io.register_observer(World, runner.world_buffer)
-            blue_full_system_proto_unix_io.register_observer(
```

Comment:
did we need to delete this?

Reply:
yup its logic has been moved into the superclass to avoid repeating code